src/clarity-angular/popover/dropdown/dropdown.module.ts

```diff
@@ -9,12 +9,14 @@ import { CommonModule } from "@angular/common";
 import { ClrCommonPopoverModule } from "../common/popover.module";
 import { ClrIconModule } from "../../icon/icon.module";
 import { DROPDOWN_DIRECTIVES } from "./index";
+import { ClrConditonalModule } from "../../utils/conditional/conditional.module";
 
 @NgModule({
     imports: [
         CommonModule,
         ClrCommonPopoverModule,
-        ClrIconModule
+        ClrIconModule,
```

Comment:
You are right, we aren't using it in the template. But since our example uses it to render the caret in the consuming side, I will export it.